src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,265 @@
+     */
+    @Prop({ reflect: true })
+    public label?: string = null;
+
+    /**
+     * A string of text that is visually placed before the value.
+     */
+    @Prop({ reflect: true })
+    public prefix?: string;
+
+    /**
+     * A piece of text or number that is the main piece of information
+     * which the component is intended to visualize.
+     */
+    @Prop({ reflect: true })
+    public value: number | string;
+
+    /**
+     * A string of text that is visually placed after the value.
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * The `title` tag of the hyperlink, which can be used to
+     * provide additional information about the link.
+     * It improves accessibility both for sighted users
+     * and users with assistive technologies.
+     */
+    @Prop({ reflect: true })
+    public linkTitle?: string = null;
+
+    /**
+     * Set to `true` if info-tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean = false;
+
+    /**
+     * The url that the info-tile leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string = null;
+
+    /**
+     * Set to `true` if info-tile is clickable.
+     */
+    @Prop({ reflect: true })
+    public clickable: boolean = false;
+
+    /**
+     * Where to load the linked URL, as the name for a browsing context:
+     * - `_self`: in the current browsing context. (Default)
+     * - `_blank`: in a new tab.
+     * - `_parent`: in the parent browsing context of the current one.
+     * If no parent, behaves as `_self`.
+     * - `_top`: the topmost browsing context (the "highest" context
+     * that's an ancestor of the current one). If no ancestors, behaves as `_self`.
+     */
+    @Prop({ reflect: true })
+    public target: '_self' | '_blank' | '_parent' | '_top' = '_self';
+
+    /**
+     * If specified, will display a notification badge
+     * on the info tile.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    /**
+     * The value of the progress bar.
+     */
+    @Prop({ reflect: true })
+    public progressValue?: number = 0;
+
+    /**
+     * The maximum value within the scale that the progress bar should visualize.
+     * Defaults to `100`.
+     */
+    @Prop({ reflect: true })
+    public progressMaxValue: number;
+
+    /**
+     * The suffix which is displayed after the `value`, must be one or two characters long.
+     * Defaults to `%`
+     */
+    @Prop({ reflect: true })
+    public progressSuffix: string = '%';
+
+    /**
+     * The prefix which is displayed before the `progressValue`,
+     * must be a few characters characters long.
+     */
+    @Prop({ reflect: true })
+    public progressPrefix?: string;
+
+    /**
+     * When set to `true`, makes the progress bar's percentage colorful.
+     * Colors change with intervals of 10%.
+     */
+    @Prop({ reflect: true })
+    public displayPercentageColors: boolean = false;
```

Comment:
I think I would prefer if these were combined to a single property instead, e.g.
```ts
interface InfoTileProgress {
    value: number;
    maxValue: number;
    prefix: string;
    suffix: string;
    displayColors: boolean;
}
```

Reply:
sounds super nice. can you show me how to do so and document it and use it. Do we have an example somewhere that I can copy?

Reply:
I added a fixup. But don't know how to use the new props in the example file. Can you add a code suggestion please?

Reply:
the fixup doesn't work right now, I'm trying to figure it out.

Reply:
works now :)

Reply:
it was not intentional 😂.  I will check what was before. and will put it back

Reply:
Everything discussed here has been done, as far as I can see, so I'm resolving this.